template-parts/content-documents.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Template part for displaying documents
+ */
+
+
+$file = get_field('document_file');
+if ($file) {
+	$link = "<a class='govuk-link' href='".$file['url']."'>".$file['title']."</a>";
+	$filesize = $file['filesize'];
+	if ($filesize < 1000) {
```

Comment:
maybe worth converting this to a function in case we need to format file size elsewhere.

This is how its formatting is done in VC. I have no preference which way but thought we could discuss.

`function convertByteSizeToHumanReadable($bytes)
{

    if ($bytes > 0) {
        $unit = intval(log($bytes, 1024));
        $units = array('B', 'KB', 'MB', 'GB');

        if (array_key_exists($unit, $units) === true) {
            return sprintf('%d %s', $bytes / pow(1024, $unit), $units[$unit]);
        }
    }

    return $bytes;
}
`

Reply:
As sugegsted by adam use size_format wordpress function

Reply:
Done - used size_format().